.swcrc.build.json

```diff
@@ -0,0 +1,20 @@
+{
+  // Note that this file is intentionally called `.swcrc.build.json`, so that if
```

Comment:
Nit: Comments aren't generally allowed in JSON. I see that SWC does allow them though.

Perhaps we could change the filename to stop editors/linters from complaining about this? The default SWC config seems to be called `.swcrc`, so perhaps we could call this `.build.swcrc`

Reply:
Sure, good point. Maybe removing the comments is fine then.

Reply:
Or we could use `.json5` as the suffix